### PR TITLE
Add delimiter option.

### PR DESCRIPTION
**Pull request by nedSaf**
#19 (7.x version)

### CODE REVIEW

**Comment by HelenaEksler** [comment]
@nader77 Let's provide the `escape` option as well. Should  solve the problem instead of https://github.com/Gizra/csv2sql/blob/8.x-1.x/csv2sql.drush.inc#L80

**Comment by HelenaEksler** [comment]
@nader77 Please, provide the D8 version as well in follow up. And provide at least one screenshot with example of the csv and the result page (using all new options).

**Comment by nedSaf** [comment]
### Drush help:

www_ _-bash_ _238x59

### Delimiter:
- Command:
  www_ _-bash_ _238x59
- File:
  account_csv_ _node__git__master_
- Resulted table:
  localhost___localhost___productivity____raw_account___phpmyadmin_4_4_10

### Enclosure and Escape
- Command:
  www_ _-bash_ _238x59
- File:
  account_csv_ _node__git__master_
- Resulted table:

localhost___localhost___productivity____raw_account___phpmyadmin_4_4_10

**Comment by nedSaf** [comment]
@HelenaEksler Ready for code review.

**Comment by nedSaf** [comment]
@amitaibu Could you review this please ?

**Comment by amitaibu** [comment]
It's your module now :) So better get a review from @HelenaEksler or some other Gizra dev.

**Comment by nedSaf** [comment]
@ordavidil Please review this :)

File: csv2sql.drush.inc [hunk 1]
```diff
@@ -38,6 +44,9 @@ function csv2sql_drush_command() {
  */
 function drush_csv2sql($csv_path) {
   $prefix = drush_get_option('prefix', '_raw');
+  $delimiter = drush_get_option('delimiter', ',');
+  $enclosure = drush_get_option('enclosure', '"');
```
Comment (ordavidil) [inline csv2sql.drush.inc:48]: The default value we passed before was `null`, what's the effect with the new default value ?

File: csv2sql.drush.inc [hunk 2]
```diff
@@ -38,6 +44,9 @@ function csv2sql_drush_command() {
...
+  $escape = drush_get_option('escape', '\\');
```
Comment (ordavidil) [inline csv2sql.drush.inc:49]: The default value we passed before was `null`, what's the effect with the new default value ?

File: csv2sql.drush.inc [hunk 3]
```diff
@@ -82,7 +91,7 @@ function drush_csv2sql($csv_path) {
         $row = array();
         foreach ($data as $delta => $value) {
           $header_col = $headers[$delta];
-          $row[$header_col] = str_replace('\"', '"', $value);
```
Comment (ordavidil) [inline csv2sql.drush.inc:85]: @nader77 
is this the effect ?  :)

**Comment by ordavidil** [comment]
@nader77 
I think we should have at least one test for this one

**Comment by nedSaf** [comment]
@ordavidil Yeah, good idea, there's no effect but I should add at least one test to it, I will look into which way is the best.

**Comment by nedSaf** [comment]
Follow-up issue for tests #29.

**Pull request merged**